templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/index.vue

```diff
@@ -217,103 +235,165 @@ export default {
+    }
+  }
+
+  &.unit-child {
+    width: 250px;
+    max-width: 25vw;
+  }
+
   .page-nav {
-    position: relative;
-    color: white;
+    color: #ffffff;
     background: #5d656c;
     padding: 2.2rem 1.5rem;
-    transform: translateY(0);
-    transition: all 0.2s ease-in-out;
     font-size: 14px;
     text-align: left;
-    z-index: 0;
     overflow-y: auto;
-    min-width: 200px;
     display: flex;
     flex-direction: column;
 
-    &.lightbox {
-      position: absolute;
-      left: 0;
-      top: 0;
-      z-index: 11;
-      border-radius: 15px 0 0 15px;
+    @media screen and (min-width: 960px) {
+      font-size: clamp(14px, 1.3vw, 16px);
+    }
 
-      &.closed {
-        background: transparent;
-        min-width: 20px;
-        max-width: 20px;
-      }
+    .page-nav-container {
+      text-align: left;
     }
 
-    &.fullscreen {
-      height: "100vh";
-      margin: -24px 24px 0 -24px;
-      z-index: 11;
+    .page-nav-content {
+      &:first-child {
+        margin-top: 9em;
+      }
 
-      &.closed {
-        @media screen and (max-width: 800px) {
-          min-width: calc(16.33px + 3rem);
-          width: calc(16.33px + 3rem);
-        }
+      .page-menu-items {
+        margin-left: 2em;
+        margin-right: -0.5em;
       }
     }
 
-    &.is-unit-child {
-      width: 250px;
-      max-width: 25vw;
+    .logout-link {
+      color: #ffffff;
     }
+  }
+}
 
-    @media screen and (min-width: 960px) {
-      font-size: calc(14px + (2 * (100vw - 960px) / 1280px - 960px));
+$slide-fade-speed: 0.3s;
+
+.slide-fade-enter-active {
+  white-space: nowrap;
+
+  &:not(.unit-child) {
+    animation: slide-fade $slide-fade-speed ease;
+
+    &.page-nav {
+      animation: slide-fade $slide-fade-speed ease;
     }
+  }
+  &.unit-child {
+    animation: slide-fade-unit-child $slide-fade-speed ease;
 
-    @media screen and (min-width: 1280px) {
-      font-size: 16px;
+    &.page-nav {
+      animation: slide-fade-unit-child $slide-fade-speed ease;
     }
+  }
+}
+.slide-fade-leave-active {
+  white-space: nowrap;
 
-    .page-nav-toggle {
-      background-color: transparent;
-      padding: 0;
-      margin-bottom: 1em;
+  &:not(.unit-child) {
+    animation: slide-fade $slide-fade-speed ease reverse;
 
-      &.fullscreen {
-        @media screen and (min-width: 801px) {
-          display: none;
-        }
-      }
+    &.page-nav {
+      animation: slide-fade $slide-fade-speed ease reverse;
     }
+  }
+  &.unit-child {
+    animation: slide-fade-unit-child $slide-fade-speed ease reverse;
 
-    .page-nav-toggle + .page-nav-content {
-      margin-top: 9em;
+    &.page-nav {
+      animation: slide-fade-unit-child $slide-fade-speed ease reverse;
     }
+  }
+}
 
-    .page-nav-container {
-      text-align: left;
-    }
+/*
+.slide-fade-enter-active,
+.slide-fade-leave-active {
+  transition: all 3s ease;
+}
+.slide-fade-enter,
+.slide-fade-leave-to {
+  min-width: 5px;
+  max-width: 5px;
+  opacity: 0;

```

Comment:
Could this commented code be removed, if safe?

Reply:
Removed.